test/onnx/test_fx_to_onnx.py

```diff
@@ -64,8 +64,8 @@ def forward(self, input):
                     torch.argmax(input, dim=1, keepdim=True),
                 )
 
-        onnx_model = fx_onnx.export(
-            ArgminArgmaxModel(), self.opset_version, model_input
+        _ = fx_onnx.export(
```

Comment:
Is it better to remove `_ = ` altogether, wdyt?

Reply:
since export has return values, I recommend that we be explicit about it